tests/unit/lms/services/jwt_test.py

```diff
@@ -57,24 +63,112 @@ def test_encode_with_secret(self, svc, jwt):
         )
         assert encoded_jwt == jwt.encode.return_value
 
+    def test_decode_lti_token(self, svc, jwt, lti_registration_service):
+        registration = factories.LTIRegistration(key_set_url="http://jwk.com")
+        lti_registration_service.get.return_value = registration
+        jwt.decode.return_value = {"aud": "AUD", "iss": "ISS"}
+
+        payload = svc.decode_lti_token(sentinel.id_token)
+
+        lti_registration_service.get.assert_called_once_with("ISS", "AUD")
```

Comment:
We seem to be missing some asserts around here about how we call `jwt.decode` and `jwt.get_unverified_header`. Does that matter?